frontend/all-in-one/src/components/ReportGen/reports.tsx

```diff
@@ -0,0 +1,377 @@
+import { useState, useEffect, type ChangeEvent, type JSX } from 'react';
+
+// Types for lists
+type Project = { id: string; name: string };
+type Staff = { id: string; email: string };
+type Department = { id: string; name: string };
+
+type FilterState = {
+  scopeType?: string;
```

Comment:
ISSUE 1: Field names use camelCase but backend expects snake_case
 Backend expects: scope_type, scope_id, start_date, end_date, time_frame
Need to convert these to snake_case before sending to API
```suggestion
  scope_type?: string;
```